usecases/auth/authorization/rbac/manager.go

```diff
@@ -31,8 +33,8 @@ type manager struct {
 	logger logrus.FieldLogger
 }
 
-func New(rbacStoragePath string, rbac rbacconf.Config, logger logrus.FieldLogger) (*manager, error) {
-	csbin, err := Init(rbac, rbacStoragePath)
+func New(rbacStoragePath string, rbac rbacconf.Config, authNconf config.Authentication, logger logrus.FieldLogger) (*manager, error) {
```

Comment:
question: what means N in authNconf?

Reply:
autheNtication is often appreviated as authN. And there is authoriZation which is appreviated as authZ

Reply:
got it!